### PR TITLE
Update KubeCon dates for 2023

### PR DESCRIPTION
**Pull request by shurup**
It's time to make the KubeCon dates relevant for this year on the index page.

By the way, shouldn't we use the full event name (_"Attend KubeCon + CloudNativeCon Europe on [this-date]"_ instead of _"Attend KubeCon Europe on [this-date]"_) or such simplification is acceptable?

### CODE REVIEW

**Comment by divya-mohan0209** [comment]
Thanks @shurup for this change! I don't have a strong preference, tbh, and am cool with either. If you want to change it, I'd gladly add the LGTM after you commit the change.

**Comment by shurup** [comment]
Personally, I like the short version (just _KubeCon_) for this page more as it's easier to read and emphasises Kubernetes itself. If there are no strict rules always requiring to use of the full event name, let's proceed with how it is now :+1:

**Comment by divya-mohan0209** [comment]
Here you go, then
/lgtm

**Comment by divya-mohan0209** [comment]
Actually, I'm sorry. Cancelling the LGTM since the KubeCon EU dates are from 18-21 April.
/lgtm cancel

File: content/en/_index.html
Comment (divya-mohan0209) [inline content/en/_index.html:46]:
```suggestion
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" button id="desktopKCButton">Attend KubeCon Europe on April 18-21, 2023</a>
```

**Comment by shurup** [comment]
Oh, it's great you noticed that! Sorry, I missed that change.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Review by Pradumnasaraf — APPROVED** [review]
Looks good.

/ltgm

**Comment by sftim** [comment]
CNCF Marketing might prefer us to leave in CloudNativeCon as part of the name. The conference does cover more than Kubernetes itself.

**Comment by shurup** [comment]
Thank you, @sftim! You made me find [this](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/program/speaker-guide/#conference-name-logos-branding). The official speaker guide is explicit on that issue:

>  You may shorten the name to just KubeCon + CloudNativeCon but you may not refer to the conference as KubeCon only. 

Thus I made a relevant change and squashed all commits we had here.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Review by divya-mohan0209 — COMMENTED** [review]
The edit is minor and the links are correctly repointed, hence approving.
/approve

**Pull request merged**
